lace/test_obj.py

```diff
@@ -188,6 +188,49 @@ def test_reading_obj_with_mtl_from_s3_url(self, mock_s3_open):
         ])
         self.assertEqual(m.materials_filepath, self.obj_with_texure_mtl)
         self.assertEqual(m.texture_filepath, self.obj_with_texure_tex)
+        self.assertIsNotNone(m.texture_image)
+
+    def test_changing_texture_filepath(self):
+        m = obj.load(self.obj_with_texure)
+        self.assertEqual(m.texture_filepath, self.obj_with_texure_tex)
+        self.assertIsNotNone(m.texture_image)
+        m.texture_filepath = None
+        self.assertIsNone(m.texture_image)
+
+    def test_texture_in_Kd_vs_Ka(self):
```

Comment:
Addressing this kind of comments is eerily easy... I keep thinking I should be doing more than copy+paste but there is basically nothing else left.
Thanks
(And you are totally right, by the way. It was basically crying to be refactored)